### PR TITLE
chore: Update feature request template

### PR DESCRIPTION
**Pull request by sithlord48**
For a feature request we only want to know about the proposed feature

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -6,85 +6,11 @@ body:
...
-      label: What problem are you trying to solve?
-      description: |
-        The most important part of problem solving is understanding the problem.
-        The more detail you provide, the better we can understand your idea.
-        “If I had only one hour to save the world, 
-        I would spend fifty-five minutes defining the problem,
-        and only five minutes finding the solution.”
-        — Albert Einstein
-    validations:
-      required: true
```
Comment (nbolton) [inline .github/ISSUE_TEMPLATE/feature_request.yml:81]: I'd like to at least keep the problem/solution format. I have found it yields much more coherent feature requests when people are guided into thinking about the problem they're trying to solve.

I can see the other fields being a bit much though.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -6,85 +6,11 @@ body:
...
-    attributes:
-      label: OS versions/distros
-      description: |
-        Please provide the version number of your operating system (OS).
-        If you're using Linux, please provide the name of the distribution.
-      placeholder: |
-        - Windows 11
-        - macOS 15
-        - Ubuntu 24.04
-        - FreeBSD 14.0
```
Comment (nbolton) [inline .github/ISSUE_TEMPLATE/feature_request.yml:50]: You sure you don't want to keep the OS but make it optional? I'm happy to let this one go if you think it's just noise.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (sithlord48) [reply .github/ISSUE_TEMPLATE/feature_request.yml:81]: Feature requests do not always have a "problem"

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (sithlord48) [reply .github/ISSUE_TEMPLATE/feature_request.yml:50]: Why do you think only ppl with some os will care about the feature>?

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (sithlord48) [reply .github/ISSUE_TEMPLATE/feature_request.yml:81]: My general thought here more mandatory fields the more friction for a feature request.

Comment (nbolton) [reply .github/ISSUE_TEMPLATE/feature_request.yml:81]: Ok, let's try it.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (nbolton) [reply .github/ISSUE_TEMPLATE/feature_request.yml:50]: You got me. Can't think of a good reason.

**Review by nbolton — APPROVED** [review]

**Pull request merged**